src/tracer.cc

```diff
@@ -33,34 +33,44 @@
 #ifdef TRITON_ENABLE_GPU
 #include <cuda_runtime_api.h>
 #endif  // TRITON_ENABLE_GPU
+#ifndef _WIN32
+namespace otlp = opentelemetry::exporter::otlp;
+namespace otel_trace_sdk = opentelemetry::sdk::trace;
+namespace otel_trace_api = opentelemetry::trace;
+namespace otel_common = opentelemetry::common;
+#endif
 
 namespace triton { namespace server {
 
 TRITONSERVER_Error*
 TraceManager::Create(
     TraceManager** manager, const TRITONSERVER_InferenceTraceLevel level,
     const uint32_t rate, const int32_t count, const uint32_t log_frequency,
```

Comment:
Shouldn't settings like rate, count, log_frequency and filepath be a part of config_map?
Here and everywhere below.

Reply:
They should, but I think it would be better to adjust it in a separate PR, since it is a significant change. For now the logic is these args are set or overrided  during the post parsing validation in `command_line_parser.cc`.